src/common/test-runtime/project.json

```diff
@@ -9,7 +9,7 @@
     "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24813-03",
     "System.Linq.Parallel": "4.0.1",
     "OpenCover": "4.6.519",
-    "ReportGenerator": "2.5.0",
+    "ReportGenerator": "2.5.9",
```

Comment:
We will at least need an update the CodeCoverage.targets file. Instead of updating the version number though I think I would go about deleting the version numbers. Then add the properties to dependencies.props in corefx so they are defined and can be picked up by both CodeCoverage.targets as well as the Xunit.Runtime.depproj file.